junit-jupiter-engine/src/test/java/org/junit/jupiter/engine/execution/ParameterResolutionUtilsTests.java

```diff
@@ -161,31 +160,33 @@ void passContextInformationToParameterResolverMethods() {
 	}
 
 	@Test
-	void invocationOfMethodsWithPrimitiveTypesIsSupported() {
+	void resolvingArgumentsForMethodsWithPrimitiveTypesIsSupported() {
 		testMethodWithASinglePrimitiveIntParameter();
 		thereIsAParameterResolverThatResolvesTheParameterTo(42);
 
-		invokeMethod();
+		Object[] arguments = resolveMethodParameters();
 
-		verify(instance).primitiveParameterInt(42);
+		assertThat(arguments).hasSize(1).containsExactly(42);
```

Comment:
```suggestion
		assertThat(arguments).containsExactly(42);
```

`hasSize()` is already covered by `containsExactly()`